### ISSUE TITLE
Unknown props with React 15.2

### ISSUE THREAD

**Issue by mikecousins** [JedWatson/react-select#1073]
react-select now throws a warning when using React 15.2:

```
warning.js:44 Warning: Unknown prop `minWidth` on <input> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in input (created by AutosizeInput)
    in div (created by AutosizeInput)
    in AutosizeInput (created by Select)
    in div (created by Select)
    in div (created by Select)
    in Select (created by Async)
    in Async (created by ComposePage)
    in div (created by ComposePage)
    in div (created by ComposePage)
    in div (created by ComposePage)
    in div (created by ComposePage)
    in ComposePage (created by RouterContext)
```

The React team says this needs to be fixed as it will break in the next major release: https://github.com/facebook/react/issues/7152#issuecomment-230273479

**Comment by reactima** [comment]
Quite disappointed by React 15.2. Has to make a deep dive into both react-select and redux-form
https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.2

Simple fix by adding .input didn't help 

`<Select
          placeholder="Which function best describes you?"
          options={props.userTypes}
          searchable={true}
          clearable={true}
          {...props.input}
          onBlur={()=>{}}
        />`

**Comment by JedWatson** [comment]
I've fixed this upstream and updated react-input-autosize, which should fix this. Will get a new npm release of react-select published as soon as I've completed testing & changelog for the fixes since the last release.

**Comment by arman-mukatov** [comment]
@JedWatson, when new npm release of "react-select" will be published?

**Comment by BorislavARC** [comment]
it's not good solving, we are waiting for right solving

**Comment by morsdyce** [comment]
Any chance of getting this fix in the 0.9 branch?

**Comment by stupidisum** [comment]
Please fix 0.9 branch for users are using allowCreate feature

**Comment by skovhus** [comment]
Any reason why this wasn't fixed for the 0.9 branch? Seems like a rather small thing to do... Any incompatabilities? Or just need help @JedWatson?

**Comment by mnieber** [comment]
I moved to `react-selectize` (not to critize `react-select`, which is great, but this fixed my immediate problem)
